db/dbformat.h

```diff
@@ -549,6 +549,10 @@ class InternalKeySliceTransform : public SliceTransform {
     return transform_->InRange(user_key);
   }
 
+  virtual bool FullLengthEnabled(size_t* len) const override {
+    return transform_->FullLengthEnabled(len);
+  }
```

Comment:
Again. I don't understand why we need it in internal key transform.

Reply:
it's not being used anymore, will delete in next upload